olive/passes/olive_pass.py

```diff
@@ -382,27 +388,24 @@ def run(
                 rank_output_path = Path(output_model_path).with_suffix("") / str(rank)
                 output_rank_model = self._run_for_config(input_rank_model, data_root, config, rank_output_path)
                 output_filepaths.append(output_rank_model.model_path)
-            return DistributedOnnxModel(output_filepaths, inference_settings=model.inference_settings)
+            output_model = DistributedOnnxModel(output_filepaths, inference_settings=model.inference_settings)
         elif isinstance(model, CompositeOnnxModel) and not self._accepts_composite_model:
             components = []
             component_names = []
             for cidx, child in enumerate(model.get_model_components()):
                 component_output_path = Path(output_model_path).with_suffix("") / str(cidx)
                 output_model_components = self._run_for_config(child, data_root, config, str(component_output_path))
```

Comment:
not related to this PR but the name `output_model_component` is confusing. since it is actually only once component.

Reply:
I think you are talking about `output_model_compont[s]`, I remove that now.

Reply:
Thanks! There’s also some in onnx_conversion pass if you don’t mind.

Reply:
Updated.